addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/VulnerabilityCache.java

```diff
@@ -104,6 +104,10 @@ private static synchronized void createSingleton() {
         }
     }
 
+    public void add(final Product p, LinkedList<CVE> vulnerabilities) {
```

Comment:
Maybe add a javadoc clarifying that this exists primarily to facilitate unit tests and isn’t really intended for other external usage.

Reply:
A javadoc-comment is added.